ISSUE_TEMPLATE

```diff
@@ -0,0 +1,6 @@
+**AS** a user
```

Comment:
This file should be in a `.github` folder to avoid cluttering. See [this](https://github.com/blog/2111-issue-and-pull-request-templates)